### PR TITLE
Feature/80 initialize user build desing documentation

### PR DESCRIPTION
**Pull request by QW3RAT**
Added the 3 initialized documentations to the deliverables folder

### CODE REVIEW

File: Deliverables/sprint-06/build-documentation.pdf.pdf
Comment (lheimbs) [inline Deliverables/sprint-06/build-documentation.pdf.pdf:1]: one too many `.pdf`

**Review by lheimbs — CHANGES_REQUESTED** [review]
I don't think the layout should be centered, it looks a bit weird to me

**Comment by QW3RAT** [comment]
Fixed request by niko and added bold headlines

**Comment by nikomall34** [comment]
It looks better now so for me its fine. But what you can do is you can just copy the md code from the wiki page and open it in vscode. Vscode support a good md to pdf converter.
Check out the attached file 
[Untitled-1.pdf](https://github.com/amosproj/amos2023ws02-pitest-ide-plugin/files/13581673/Untitled-1.pdf)

**Pull request merged**